src/main/java/mate/academy/service/CinemaHallService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.service;
+
+import java.util.List;
+import mate.academy.model.CinemaHall;
+
+public interface CinemaHallService {
+
+    CinemaHall add(CinemaHall cinemaHall);
+
+    CinemaHall get(Long id);
```

Comment:
The `get()` method should return an `Optional<CinemaHall>` instead of `CinemaHall`. This is important for handling potential null values safely, as specified in the task requirements.